src/ReactionCIMEApp.tsx

```diff
@@ -18,6 +18,7 @@ import { FilterTabPanel } from './Overrides/FilterTabPanel/FilterTabPanel';
 import { PacoTabPanel } from './Overrides/PacoTabPanel/PacoTabPanel';
 import { AddRegionExceptionMenuItem } from './Overrides/ContextMenu/AddRegionException';
 import { SetFiltersToItemFeatures } from './Overrides/ContextMenu/SetFiltersToItemFeatures';
+import 'react-data-grid/lib/styles.css';
```

Comment:
@ginihumer `react-data-grid` does not contain this file: https://unpkg.com/browse/react-data-grid@7.0.0-beta.16/lib/

However, if we would upgrade [here](https://github.com/jku-vds-lab/projection-space-explorer/blob/develop/package.json#L40) to https://unpkg.com/browse/react-data-grid@7.0.0-beta.17/lib/ it would exist. 
Currently, this file is part of the bundle.js itself, i.e. not importable (webpack should throw an error when building).

Reply:
Edit: Looking twice, I see that we are actually overriding this dependency and use beta.20 instead: https://github.com/jku-vds-lab/reaction-cime/blob/develop/package.json#L91

@dvmoritzschoefl I would strongly suggest that we upgrade the version in PSE, test in CIME and Reaction-CIME, and remove the resolution from this package. This is for sure becoming a problem in the long run.

Reply:
@puehringer I recall that there was a problem with the react version since starting with beta (17 I think) it will force React 18

Reply:
That's correct: https://github.com/adazzle/react-data-grid/blob/v7.0.0-beta.17/package.json#L113-L114
However, we are forcing React to be version 16 via resolutions. Maybe react-data-grid doesn't use an 18 only features, because it worked till now in Reaction-CIME.

Reply:
@ginihumer Could you try removing this style import, and also remove the `beta.20` resolution in the package.json, and then run `yarn` to reinstall `beta.16`, then run `yarn start` again and check if it works? That would streamline the version w/ PSE and allow PSE to manage what happens.

Reply:
ok, works now